### PR TITLE
Add fallback to failed requests to generate the output file.

### PR DESCRIPTION
**Pull request by kiliczsh**
I have returned empty array for ignoring the failures.

### CODE REVIEW

**Review by BYK — APPROVED** [review]
I guess this is fine 😅

File: main.py [hunk 1]
```diff
@@ -38,12 +38,15 @@ async def fetch(self, url) -> list[dict]:
                     await asyncio.sleep(0.1 * 2**try_no)
                 elif isinstance(e, aiohttp.ClientResponseError):
                     logging.error("%s failed with code %s", url, e.status)
-                    raise e
+                    if e.status == 520:
+                        raise RuntimeError("Cloudflare error message indicating that the origin web server received an invalid or incorrectly interpreted request, resulting in an empty response.")
```
Comment (BYK) [inline main.py:42]: Are you sure you don't want to retry on 520?

**Comment by kiliczsh** [comment]
> I guess this is fine 😅 

I think my ip address is blocked 😄

**Review by egeakman — APPROVED** [review]

File: main.py (hunk 1 above)
Comment (BYK) [reply main.py:42]: Eh, I would have expected a [`403`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/403) for that.

Comment (BYK) [reply main.py:42]: Sounds good. Cloudflare's own documentation say it is a problem with the origin server, not a block: https://community.cloudflare.com/t/community-tip-fixing-error-520-web-server-is-returning-an-unknown-error/44205

**Comment by shazar** [comment]
I came across this after I published my scraped dataset on Github. Due to Cloudflare and rate limiting, I think scraping this from a single computer is hard and time consuming.

If you need the full dataset in Sqlite format, you can use this: https://github.com/shazar/2023ovodata

**Comment by kiliczsh** [comment]
> I came across this after I published my scraped dataset on Github. Due to Cloudflare and rate limiting, I think scraping this from a single computer is hard and time consuming.
> 
> If you need the full dataset in Sqlite format, you can use this: https://github.com/shazar/2023ovodata

Hi @shazar, I added sql script in zip file thank you so much.

**Pull request closed without merging**